extras/systemd/foreman.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman
+Documentation=https://theforeman.org
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=simple
+User=$FOREMAN_USER
+TimeoutSec=300
+WorkingDirectory=/usr/share/foreman
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_BIND"
+Environment=FOREMAN_ENV=production FOREMAN_PORT=3000 FOREMAN_BIND=0.0.0.0 FOREMAN_USER=foreman
+EnvironmentFile=-/etc/sysconfig/foreman
```

Comment:
This is a RH thing to do. Debian has `/etc/default`. IIRC systemd recommended using drop in files instead. In foreman-proxy we left it out and then added it back in packaging. IMHO that's still the best thing to do.

Reply:
Dropped this line